sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1096,6 +1106,7 @@ def __init__(
     self.failed_rows_metric = Metrics.distribution(
         self.__class__, "rows_failed_per_batch")
     self.bigquery_wrapper = None
+    self._latency_logging_frequency = latency_logging_frequency or 180
```

Comment:
add units to variable name
i.e.

_latency_logging_frequency_msec

Reply:
done.